gapic/src/main/com/google/gapid/models/Perfetto.java

```diff
@@ -95,8 +95,9 @@ public void onCaptureLoaded(Loadable.Message error) {
             transformAsync(withStatus("Querying GPU info...", queryGpu(data)), $3 ->
               transformAsync(withStatus("Querying Frame info...", queryFrame(data)), $4 ->
                 transformAsync(withStatus("Querying counters...", queryCounters(data)), $5 ->
-                  transform(withStatus("Enumerating tracks...", enumerateTracks(data)), $6 ->
-                    data.build()))))));
+                  transformAsync(withStatus("Querying counter groups...", queryCounterGroups(data)), $6 ->
```

Comment:
I don't think this needs it's extra section. Getting the group information should just be part of the counters.

Reply:
Done.